cfgov/unprocessed/apps/owning-a-home/js/explore-rates/Slider.js

```diff
@@ -0,0 +1,113 @@
+// Required modules.
+const atomicHelpers = require( '../../../../js/modules/util/atomic-helpers' );
+const standardType = require( '../../../../js/modules/util/standard-type' );
+
+const $ = require( 'jquery' );
```

Comment:
use import here

Reply:
What would this look like? Try changing it in `rate-checker.js` and running `gulp build`. Do you get an error in the dev console? If not what's the import statement you used look like?